### PR TITLE
Adds configuration to build docs and publish on packer.io

### PR DESCRIPTION
**Pull request by scottford-io**
THIS IS A DRAFT PR. DO NOT MERGE UNTIL REVIEWED. NEEDS WORK

Signed-off-by: Scott Ford <scott@scottford.io>

### CODE REVIEW

File: docs/mondoo.mdx [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+description: The Mondoo packer provisioner will scan builds for vulnerabilities and misconfigurations by executing security policies-as-code.
```
Comment (preslavgerchev) [inline docs/mondoo.mdx:2]: @scottford-io Please have a look at this description, I just threw something in quickly, feel free to improve!

**Comment by preslavgerchev** [comment]
Needs commits squashing + DCO, let's do that after docs have been looked at

File: docs/mondoo.mdx (hunk 1 above)
Comment (scottford-io) [reply docs/mondoo.mdx:2]:
```suggestion
description: The Mondoo packer provisioner scans machine-image builds for vulnerabilities and misconfigurations by executing security policies-as-code.
```

@preslavgerchev small suggestion

**Comment by scottford-io** [comment]
@nywilken do you see anything missing from this PR?

**Pull request merged**